udata_front/frontend/__init__.py

```diff
@@ -77,3 +77,11 @@ def test_500():
     if app.config.get('DEBUG_TOOLBAR'):
         from flask_debugtoolbar import DebugToolbarExtension
         DebugToolbarExtension(app)
+
+    # Security override init
+    from udata.auth import security
+    from udata_front.forms import ExtendedRegisterForm
+
+    with app.app_context():
+        security._state.register_form = ExtendedRegisterForm
+        security._state.confirm_register_form = ExtendedRegisterForm
```

Comment:
Do we need both `confirm_register_form` and `register_form`?

Reply:
Both are required because one or the other is used depending of `SECURITY_CONFIRMABLE` value. This was already fixed for udata https://github.com/opendatateam/udata/pull/2588